google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/GrpcResultSetTest.java

```diff
@@ -665,4 +665,161 @@ private void verifySerialization(
     Struct copy = reserialize(row);
     assertThat(row).isEqualTo(copy);
   }
+
+  @Test
+  public void getBoolean() {
+    consumer.onPartialResultSet(
+        PartialResultSet.newBuilder()
+            .setMetadata(makeMetadata(Type.struct(Type.StructField.of("f", Type.bool()))))
+            .addValues(Value.bool(true).toProto())
+            .addValues(Value.bool(false).toProto())
+            .build());
+    consumer.onCompleted();
+    resultSet.next();
+    assertThat(resultSet.getBoolean(0));
+    resultSet.next();
+    assertThat(!resultSet.getBoolean(0));
+  }
+
+  @Test
+  public void getDouble() {
+    consumer.onPartialResultSet(
+        PartialResultSet.newBuilder()
+            .setMetadata(makeMetadata(Type.struct(Type.StructField.of("f", Type.float64()))))
+            .addValues(Value.float64(Double.MIN_VALUE).toProto())
+            .addValues(Value.float64(Double.MAX_VALUE).toProto())
+            .build());
+    consumer.onCompleted();
+
+    resultSet.next();
+    Double d = resultSet.getDouble(0);
+    assertThat(d.equals(Double.MIN_VALUE));
```

Comment:
is there assertEquals ?

Reply:
Same.

Reply:
this is pending as per Michael's comment of .isEqualTo check

Reply:
Missed it, thanks.